karapace/config.py

```diff
@@ -86,14 +86,13 @@ def write_config(config_path: str, custom_values: Dict[str, Union[str, int, bool
         fp.write(json.dumps(custom_values))
 
 
-def read_config(config_path: str) -> Dict[str, Union[str, int, bool]]:
-    with open(config_path, "r") as cf:
```

Comment:
I changed the command line parser to use the built-in `FileType`. Which will verify if the file exists and give back a opened handler. The code path with a file name is not necessary any more after this change.

Reply:
Neat!